src/AddressRegistry.Api.Oslo/paket.references

```diff
@@ -0,0 +1,13 @@
+AspNetCore.HealthChecks.SqlServer
+
+Be.Vlaanderen.Basisregisters.Api
+Be.Vlaanderen.Basisregisters.EventHandling.Autofac
```

Comment:
Can you check if this package is still needed?

Reply:
not needed